testkit-backend/backend.go

```diff
@@ -1147,21 +1147,21 @@ func (b *backend) consumeBookmarks(bookmarkManagerId string) func(string, neo4j.
 		for {
 			b.process()
 			if _, found := b.consumedBookmarks[id]; found {
-				return
+				return nil
 			}
 		}
 	}
 }
 
 type testkitBookmarkSupplier struct {
-	supplierFn func(...string) neo4j.Bookmarks
+	supplierFn func(...string) (neo4j.Bookmarks, error)
```

Comment:
```suggestion
	supplier func(...string) (neo4j.Bookmarks, error)
```

nitus pickus maximus :upside_down_face:

Reply:
Fixed in e2299cc5f46f2f6acd623af39a96bc950994ef70